packages/web/.eslintrc

```diff
@@ -0,0 +1,59 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "airbnb",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "settings": {
+    "import/extensions": [".js",".jsx",".ts",".tsx"],
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts",".tsx"]
+    },
+    "import/resolver": {
+      "node": {
+        "extensions": [".js",".jsx",".ts",".tsx"]
+      }
+    }
+  },
+  "rules": {
+    "@typescript-eslint/no-unused-vars": "error",
+    "@typescript-eslint/no-useless-constructor": "error",
+    "import/extensions": [
+      "error",
+      "ignorePackages",
+      {
+        "js": "never",
+        "jsx": "never",
+        "ts": "never",
+        "tsx": "never"
+      }
+    ],
+    "react/jsx-filename-extension": [
+      1,
+      {
+        "extensions": [
+          ".js",
+          ".jsx",
+          ".ts",
+          ".tsx"
+        ]
+      }
+    ],
+    "react/jsx-one-expression-per-line": "off",
+    "jsx-quotes": ["error", "prefer-double"],
+    "quotes": ["error", "single"],
+    "comma-dangle": ["error", {
+      "arrays": "always-multiline",
+      "objects": "always-multiline",
+      "imports": "never",
+      "exports": "never",
+      "functions": "never"
```

Comment:
Can you just omit the watchAll flag?
```
"pre-push": "yarn ts:check && yarn test"
```

Reply:
But in this case, we will need to add a comma in function arguments. 
Got example
![image](https://user-images.githubusercontent.com/26768476/84881158-24f76f00-b096-11ea-8358-7ffaeabaddf8.png)
I think it isn't good

Reply:
It's pretty good in my opinion, for a reason above. It's just as good as:

```
import {
  Header,
  Body,
  Footer,
} from './components'
```
Same for functions, arrays, objects, and functions. See:

https://eslint.org/docs/rules/comma-dangle

![image](https://user-images.githubusercontent.com/16776066/84882045-57559c00-b097-11ea-97da-4a31a7f95c06.png)

Reply:
I mean it's default for a reason, and I believe it is a good style to get used to.
But we might as well just have a quick vote, Team.

Reply:
+ for trailing comma rule

Reply:
Cannot argue. Let's keep trailing comma